Automattic/wp-calypso
# Gravatar: Fix toggle button overlap

* Closes https://github.com/Automattic/wp-calypso/issues/74598

## Proposed Changes

* This PR fixes the overland between the toggle button and the "Hide my photo and Gravatar profile" text on `https://wordpress.com/me`.

## Testing Instructions

1. Navigate to `https://wordpress.com/me`
2. Check that the toggle by the text "Hide my photo and Gravatar profile"  does not overlap the text:

<img width="1442" alt="Screenshot 2023-04-06 at 1 47 26 PM" src="https://user-images.githubusercontent.com/25575134/230367622-db3cc57a-53c3-45fd-b271-446876bfa253.png">

3. Check the display from the mobile browser (using Chrome dev tools):

<img width="837" alt="Screenshot 2023-04-06 at 1 48 06 PM" src="https://user-images.githubusercontent.com/25575134/230367778-bb2a9638-bfad-480d-b8ba-592adc912925.png">

Screenshot from before the changes:

<img width="1461" alt="Screenshot 2023-04-06 at 1 49 21 PM" src="https://user-images.githubusercontent.com/25575134/230368366-5f3b27e3-911b-4a5f-9896-beae3dadca64.png">


## Pre-merge Checklist
